data/seeds/001_profiles.js

```diff
@@ -1,21 +1,50 @@
-const faker = require('faker');
+// const faker = require('faker');
```

Comment:
Is the commented out code needed for testing or anything else?

Reply:
I removed the commented out boilerplate code on the migration file, but I wanted to keep this commented out code for the time being because I want to adapt it to generate many new users for our seed file using our version of the profiles table. If we don't end up using it later, I'll delete it completely.

Reply:
Perfect!! This works, thank you for that insight.